libretro-cores/libretro-nestopia.json

```diff
@@ -23,7 +23,8 @@
                 {
                     "type": "file",
                     "path": "nestopia.libretro",
-                    "dest-filename": "libretro/nestopia.libretro"
+                    "dest": "libretro",
+                    "dest-filename": "nestopia.libretro"
```

Comment:
This one will fail on line 15, need to adjust it there as well. I fixed this in Highscore a while ago, though Citra doesn't build atm.

Reply:
line 15 execute inside `libretro`, this file is copied on `libretro` (dest).

I could have removed dest-filename though since it's now the same.

Reply:
Oh right, nevermind me. I need to sync the `dest` change into highscore as well then, when I find energy to get back to it.

Reply:
note: I was wondering why this failed now and not on March 24 where last build happened and now I realise that it was March 24 2021. And since flatpak-builder has started rejected `/` in dest-filename.

Reply:
this change is meant to be neutral, just to fix the syntax that is now an error.